comp/logs/agent/agent.go

```diff
@@ -254,3 +268,36 @@ func (a *agent) GetMessageReceiver() *diagnostic.BufferedMessageReceiver {
 func (a *agent) GetPipelineProvider() pipeline.Provider {
 	return a.pipelineProvider
 }
+
+// func (a *agent) FlareProvider() flaretypes.Provider {
+// 	return flaretypes.NewProvider(a.fillFlare)
+// }
+
+func (a *agent) fillFlare(fb flaretypes.FlareBuilder) error {
+	tailers := a.tracker.All()
+
+	for _, tailer := range tailers {
+		fmt.Println(tailer.GetId())
+		fmt.Println(tailer.GetType())
+		fmt.Println(tailer.GetInfo())
+		fmt.Println()
+	}
+
+	fb.AddFileFromFunc("logs_file_permissions.log", func() ([]byte, error) {
```

Comment:
@soberpeach Could we add tests around this? https://github.com/DataDog/datadog-agent/tree/main/test/new-e2e/tests/agent-subcommands/flare